### PR TITLE
StorIOContentReslver fix for observing changes of Uris on Android API < 16

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
@nikitin-da PTAL, I'll also invite all team from my project :)

### CODE REVIEW

**Comment by artem-zinnatullin** [comment]
Fixes #549.

**Comment by karlicoss** [comment]
ok!

**Comment by nikitin-da** [comment]
LGTM :+1:

**Comment by StanisVS** [comment]
ok

**Comment by EvaGL** [comment]
looks ok for me

File: storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/impl/RxChangesObserver.java [hunk 1]
```diff
@@ -27,37 +28,67 @@ private RxChangesObserver() {
...
+                        public void onChange(boolean selfChange, Uri uri) {
+                            subscriber.onNext(Changes.newInstance(uri));
+                        }
+                    };
 
-                    @Override
-                    public void onChange(boolean selfChange, Uri uri) {
-                        subscriber.onNext(Changes.newInstance(uri));
+                    for (Uri uri : uris) {
+                        contentResolver.registerContentObserver(
```
Comment (thevery) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/impl/RxChangesObserver.java]: a bit inconsistent formatting with https://github.com/pushtorefresh/storio/pull/550/files#diff-aaef05f8096c3760f578dccfd884908cR82

Comment (artem-zinnatullin) [inline storio-content-resolver/src/main/java/com/pushtorefresh/storio/contentresolver/impl/RxChangesObserver.java]: Jesus, mr formatting here! I guess we'll just apply codestyle to whole project at some point :)

**Pull request merged**